src/replication.thrift

```diff
@@ -478,6 +478,21 @@ struct query_disk_info_response
     4:list<disk_info> disk_infos;
 }
 
+// This request is sent from clent to replica_server.
+struct migrate_replica_request 
+{
+    1:dsn.rpc_address node;
```

Comment:
```suggestion
```

For client, the tcp destination is the "node". For server, the address of itself is the "node".
So both sides are aware of the "node", you don't have to add it in the request.